src/test/java/UserTableTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import quest.com.entity.User;
+import quest.com.repository.UserTable;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UserTableTest {
+    UserTable userTable;
+
+    @BeforeEach
+    void init() throws InvocationTargetException, InstantiationException, IllegalAccessException, NoSuchMethodException {
+        Constructor<?> userTableConstructor = UserTable.getInstance().getClass().getDeclaredConstructor();
```

Comment:
We should not use Reflection API for testing. Using reflection APi in testing means that code has smell.